### PR TITLE
Fix the error message to show how to leave out optional section

### PR DESCRIPTION
**Pull request by Hibeom0929**
also fixed some minor words to have a consistent format.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Address.java [hunk 1]
```diff
@@ -11,7 +11,9 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, "
+            + "and it should not be blank."
+            + "If you want to not specify the address, leave out a/ADDRESS entirely.";
```
Comment (E0735389) [inline src/main/java/seedu/address/model/person/Address.java:16]: Is this statement correct for `edit` command?

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 2]
```diff
@@ -9,7 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric.\n"
+            + "If you want to not specify the tag, leave out t/TAG entirely.";
```
Comment (E0735389) [inline src/main/java/seedu/address/model/tag/Tag.java:13]: Shouldn't this say "nonempty and alphanumeric" then? But what about `edit` command...

File: src/main/java/seedu/address/model/person/Address.java (hunk 1 above)
Comment (Hibeom0929) [reply src/main/java/seedu/address/model/person/Address.java:16]: This one will not be shown for edit command as address accept all.

File: src/main/java/seedu/address/model/tag/Tag.java (hunk 2 above)
Comment (Hibeom0929) [reply src/main/java/seedu/address/model/tag/Tag.java:13]: That's right, but I'm afraid that "nonempty" could be misunderstood as we can't have no tag by users.

Comment (E0735389) [reply src/main/java/seedu/address/model/tag/Tag.java:13]: Yes, but seeing "tag names should be alphanumeric" for a command `edit 1 t/ t/x` is extremely confusing.

Comment (Hibeom0929) [reply src/main/java/seedu/address/model/tag/Tag.java:13]: Hmm, then how can I inform them how to not specify it when they add this...

**Pull request closed without merging**